pset-iterators.js

```diff
@@ -170,7 +207,20 @@ called `shortTermPresidents`, declared with const
 console.log('Problem 6:')
 
 // Add your code below this line
+const shortTermPresidents = []
+
+presidents.forEach(function(element){
+  let date1 = new Date (element.took_office)
+  let date2 = new Date (element.left_office)
+
+  let time = (date2 - date1)/1000/60/60/24
```

Comment:
@kertcodes interesting approach, this definitely results in a more precise calculation